shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -146,6 +160,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
                 if (object.textSelectionBase != -1 && object.textSelectionExtent != -1) {
                     result.setTextSelection(object.textSelectionBase, object.textSelectionExtent);
                 }
+                result.setLiveRegion(View.ACCESSIBILITY_LIVE_REGION_POLITE);
```

Comment:
This probably deserves an explaining comment for the next guy looking at this code.

Reply:
Added a comment